sssom/parsers.py

```diff
@@ -175,9 +175,22 @@ def _address_multivalued_slot(k: str, v: str) -> Union[str, List[str]]:
 
 def _init_mapping_set(meta: Optional[MetadataType]) -> MappingSet:
     if meta is not None:
-        return MappingSet(
-            mapping_set_id=meta["mapping_set_id"], license=meta["license"]
-        )
+        if all(["mapping_set_id", "license"]) in meta.keys():
```

Comment:
Just curious, do you like this way to if..else more than:

```
license=DEFAULT_LICENSE
mapping_set_id=DEFAULT_MAPPING_SET_ID
if meta is not None:
        if "mapping_set_id" in meta.keys():
                mapping_set_id=meta["mapping_set_id"]
        if "license" in meta.keys():
                license = meta["license"]
return MappingSet(
                mapping_set_id=mapping_set_id, license=license
            )
```

Reply:
This is better! I'll change it.